api/units.go

```diff
@@ -76,6 +76,7 @@ func (ur *unitsResource) set(rw http.ResponseWriter, req *http.Request, item str
 	}
 
 	var su schema.Unit
+	newUnit := false
```

Comment:
It's easier to find the definition if it's close to the use, end vice versa.

It's not just Go: all modern languages I know allow this -- including even C99...

More to the point, it appears to be a pattern used throughout the existing fleet code -- so you need a good reason for *not* following it.